package.json

```
@@ -6,7 +6,8 @@
   "module": "dist/index.es.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
+    "test": "jest --watch",
```

Maybe this should be something like `test:watch`?
```suggestion
    "test:watch": "jest --watch",
```